<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易编译器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="gram3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>词法分析正规文法</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="gram3in">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="cpp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分析源码</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cppin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tokenout">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="token">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>词法分析结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始分析</string>
    </property>
   </widget>
   <widget class="QLabel" name="gram2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>语法分析二型文法</string>
    </property>
   </widget>
   <widget class="QLabel" name="trans">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>语法分析结果</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="gram2in">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="transout">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭窗口</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
